From addb383b375fd0db07eacf8a9f5d9b4e23343855 Mon Sep 17 00:00:00 2001
From: Steffen Maier <maier@linux.ibm.com>
Date: Fri, 26 May 2023 19:25:41 +0200
Subject: [PATCH 14/23] docs(dracut.cmdline): generalize description of rd.znet

As of the preceding commit ("feat(znet): use zdev for consolidated device
configuration"), rd.znet is no longer specific to RHEL/Fedora.

Signed-off-by: Steffen Maier <maier@linux.ibm.com>
---
 man/dracut.cmdline.7.asc | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/man/dracut.cmdline.7.asc b/man/dracut.cmdline.7.asc
index c31dbfc6..e6b8d988 100644
--- a/man/dracut.cmdline.7.asc
+++ b/man/dracut.cmdline.7.asc
@@ -1018,9 +1018,12 @@ rd.zfcp=0.0.4000
 ZNET
 ~~~~
 **rd.znet=**__<nettype>__,__<subchannels>__,__<options>__::
-    The whole parameter is appended to /etc/ccw.conf, which is used on
-    RHEL/Fedora with ccw_init, which is called from udev for certain
-    devices on z-series.
+    Activates a channel-attached network interface on s390 architecture.
+    <nettype> is one of: qeth, lcs, ctc.
+    <subchannels> is a comma-separated list of ccw device bus-IDs.
+    The list consists of 3 entries with nettype qeth, and 2 for other nettype.
+    <options> is a comma-separated list of <name>=<value> pairs,
+    where <name> refers to a device sysfs attribute to which <value> gets written.
     rd.znet can be specified multiple times on the kernel command line.
 
 **rd.znet_ifname=**__<ifname>__:__<subchannels>__::
-- 
2.42.0

